db-connect.js

```diff
@@ -0,0 +1,10 @@
+import mongoose from 'mongoose';
+
+ 
+
+//const Mongo_URL = "mongodb://localhost/users_db";
+const Mongo_URL = "mongodb+srv://shinhee:*******@shinhee.fglsw.mongodb.net/myFirstDatabase?retryWrites=true&w=majority";
```

Comment:
This connection string here will not work, if the password is missing. 
Or did you really choose ******** as password? :smile:

Reply:
thanks Robert for pointing this out. i now can use dotenv, so this won't happen again. :D